cypress/pages/Index.js

```diff
@@ -0,0 +1,42 @@
+class IndexPage{
+
+    constructor(){
+        this.btnSignIn = ".login";
+        this.categories = ".sf-menu";
+        this.btnCart = "Cart";
+        this.searchInput = "#search_query_top";
+        this.btnSearch = ".button-search";
+        this.btnList = ".icon-th-list";
+    }
+
+    clickBtnSignIn = () =>{
```

Comment:
For consistency, methods and functions should be named like this form:

clickBtnSignIn(){

}

Only in the case we have to pass a variable referencing a method/function as argument for another function/method this will be valid.